test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -371,10 +372,13 @@ public void test() throws IOException
                                                                         .add(this::selectTokenRange))
                                   .addIf(State::hasEnoughMemtable, StatefulASTBase::flushTable)
                                   .addIf(State::hasEnoughSSTables, StatefulASTBase::compactTable)
+                                  .addAllIf(BaseState::allowRepair, b -> b.add(StatefulASTBase::incrementalRepair)
+                                                                          .addIf(BaseState::isConsistent, StatefulASTBase::previewRepair))
                                   .addIf(State::allowNonPartitionQuery, this::nonPartitionQuery)
                                   .addIf(State::allowNonPartitionMultiColumnQuery, this::multiColumnQuery)
                                   .addIf(State::allowPartitionQuery, this::partitionRestrictedQuery)
                                   .destroyState(State::close)
+                                  .commandsTransformer(LoggingCommand.factory())
```

Comment:
since repairs can take awhile, its good to log whats going on so we know how long the repairs are taking if things go wrong.  In paxos v2 they were taking 3m vs the < 1s normally expected, this was due to config issue (these tests bump the write timeouts to 3m, which caused repair to take 3m as it waits based off those configs)...